developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -154,6 +154,17 @@ Now we return to our application to deploy and interact with the module on the A
   </TabItem>
 </Tabs>
 
+Note: to initialize module, you can write a `init_module` function. This private function is executed automatically when the module is published.
+The format of this function has to be private, no return value and only take signer or signer reference as parameters.  Here is an example:
+```asm
+ fun init_module(creator: &signer) {
+        move_to(
+            creator,
+            ModuleData { global_counter: 0 }
+        );
+    }
+```
+
```

Comment:
:::note
Note: To initialize module, you can write a `init_module` function. This private function is executed automatically when the module is published. This `init_module` function must be private, it should only take signer or signer reference as parameters, and it should not return any value.  See below an example:
:::

Reply:
It automerged already, I see. I will make the above change later today.